tests/scripts/kubeadm.sh

```diff
@@ -5,9 +5,9 @@ scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 tarname=image.tar
 tarfile="${WORK_DIR}/tests/${tarname}"
 
-export KUBE_VERSION=${KUBE_VERSION:-"v1.8.5"}
+export KUBE_VERSION=${KUBE_VERSION:-"v1.13.1"}
 
-if [[ $KUBE_VERSION == v1.12* ]] ; then
+if [[ $KUBE_VERSION != v1.10* && $KUBE_VERSION != v1.11* ]] ; then
```

Comment:
So Kubernetes v1.10.x and v1.11.x have a problem during `kubeadm` start?

Reply:
In 1.12 the command line option changed, so in 1.10 and 1.11 it will fall to the `else` condition. The way this condition is working, eventually the obsolete option will go away when we stop supporting older than 1.11.